generators/app/templates/_templates/field/field.html.twig

```diff
@@ -41,9 +41,6 @@
   set classes = [
     'field',
     'field--name-' ~ field_name|clean_class,
```

Comment:
Can we add another class that will specify for which bundle the field is? For example, a body field in the Article content type will have `field--body-article` or a link field in a Card block type will have `field--link-card`
This can be useful if we want different styling for the same field but different bundles/entity types without the need to add a parent class for it.